src/graphql/VariantFragment.graphql

```diff
@@ -1,5 +1,6 @@
 fragment VariantFragment on ProductVariant {
   id
+  availableForSale
```

Comment:
we should also remove `available` here since it's deprecated, and `availableForSale` replaces it